src/swift/DispatchStubs.cc

```diff
@@ -11,11 +11,11 @@
 //===----------------------------------------------------------------------===//
 
 #include <dispatch/dispatch.h>
-#include <objc/runtime.h>
 #include <stdio.h>
 
 #define DISPATCH_RUNTIME_STDLIB_INTERFACE __attribute__((__visibility__("default")))
 
+#if 0
```

Comment:
The suffix probably could be left as .mm or .m so it is the same on Darwin, and this could be flagged with an #if USE_OBJC instead of #if 0